src/assets/translations/en/main.json

```diff
@@ -663,6 +664,7 @@
       "rate": "This is the expected rate for this trade pair.",
       "noRateAvailable": "This is often due to very limited or no liquidity on the trade pair.",
       "minerFee": "This is the fee to process the transaction on the blockchain, this is not a fee charged by ShapeShift.",
+      "donation": "Help support ShapeShift and keep us up and running.",
```

Comment:
@reallybeard @willyogo the aggressiveness of this copy will dramatically impact conversions on this. as a follow up PR we should consider showing a modal with a longer explanation of why they should donate with some strong dark patterns maximize revs here